UserInterface.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace HwCreateGame
+{
+    public class UserInterface
+    {
+        private ContactList contactList;
+        private FileHandler fileHandler;
+        private bool canWriteToFile;
+
+        public UserInterface(ContactList contactList, FileHandler fileHandler, bool canWriteToFile)
+        {
+            this.contactList = contactList;
+            this.fileHandler = fileHandler;
+            this.canWriteToFile = canWriteToFile;
+        }
+
+        // Метод обработки команд пользователя
+        public void ProcessUserCommands()
+        {
+            while (true)
+            {
+                Console.WriteLine("Введите команду (1 - Добавить, 2 - Найти, 3 - Отобразить список, 4 - Выйти, 5 - Удалить):");
+                string command = Console.ReadLine();
+
+                switch (command.ToLower())
+                {
+                    case "1":
+                        if (canWriteToFile)
```

Comment:
Why interface should care about that? Who should know whether writing is forbidden? How that something should be communicated to the interface layer? Exceptions, some operation result object?